src/services/app-config-service.ts

```diff
@@ -41,6 +41,28 @@ export class AppConfigService {
     return this.validateIsHTTPSUrl('TXMA_QUEUE_URL');
   }
 
+  public get userId() {
+    return process.env['USER_ID'];
+  }
+
+  public get baseUrl() {
+    return this.validateConfiguration('BASE_URL');
+  }
+
+  public get endpoint() {
+    const endpoint = this.validateConfiguration('END_POINT');
+    const prefix = endpoint.startsWith('/') ? '' : '/';
+    return prefix + endpoint;
+  }
+
+  public get queryParameters() {
+    return process.env['QUERY_PARAMETERS'];
+  }
+
+  public get httpRequestMethod() {
+    return this.validateConfiguration('HTTP_REQUEST_METHOD');
+  }
+
   /**
```

Comment:
Do we want this lambda to use the app config service to access the environment variables? Or would it be better to keep this lambda completely separate from the main solution?

Reply:
separate imho